css/types/color.json

```diff
@@ -151,6 +151,56 @@
             }
           }
         },
+        "color": {
+          "__compat": {
+            "description": "Profiled color values <code>color()</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color_value/color()",
+            "spec_url": "https://drafts.csswg.org/css-color/#color-function",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": {
+                "version_added": "15"
+              },
```

Comment:
Hasn't

> Specifying profiled colors: the color() function

been supported since Safari 10.1?

Per
- https://github.com/Fyrd/caniuse/issues/3134
- https://webkit.org/blog/7477/new-web-features-in-safari-10-1/#:~:text=css%20wide-gamut%20colors

Or did something change or the like…?

Reply:
Yes, support did change. While `color()` function support was initially added in Safari 10.1, the only supported color profile at the time was `display-p3`. Now all of the predefined color profiles are supported.

This can be updated to reflect that.

Reply:
I see!

For <https://caniuse.com/css-color-function> my first thought is to downgrade Safari <= 14 to partial with a note like "only supports `display-p3`" and start full support with 15.

But I'll observe how it will be handled for MDN here :)

Reply:
> While `color()` function support was initially added in Safari 10.1, the only supported color profile at the time was `display-p3`.

If this was the case, then you'll need to do two support statements. One from 15 onwards, without any notes, and another from 10.1 to 15 with a note saying that only `display-p3` is supported.